src/main/java/net/snowflake/client/core/SFArrowResultSet.java

```diff
@@ -116,6 +119,7 @@
     session.setWarehouse(resultSetSerializable.getFinalWarehouseName());
     treatNTZAsUTC = resultSetSerializable.getTreatNTZAsUTC();
     formatDateWithTimezone = resultSetSerializable.getFormatDateWithTimeZone();
+    useWallclockTime = resultSetSerializable.useWallclockTime();
```

Comment:
Is this line needed? I guess we have initialized useWallclockTime in another constructor?